app/components/MintBTC.tsx

```diff
@@ -58,44 +58,73 @@ function Fee({ fee, youReceive }: ExchangeRateProps) {
 	);
 }
 
+interface MinBTCForm {
+	numberOfBTC: number;
+	suiAddress: string;
+}
+
 interface MintBTCProps {
 	availableBalance: number;
 	suiAddress: string;
 }
 
 export function MintBTC({ availableBalance, suiAddress }: MintBTCProps) {
-	const handleDeposit = useCallback(() => {
-		// Handle deposit logic here
-		console.log("Depositing with Sui Address:", suiAddress);
-	}, [suiAddress]);
+	const mintBTCForm = useForm<MinBTCForm>({
+		defaultValues: {
+			// TODO: make it dynamic
+			numberOfBTC: 1,
```

Comment:
due to rounding errors, this must be a text

Reply:
Yes, it should be text.